src/app/ngrx/reducers/selectors/export.selectors.spec.ts

```diff
@@ -34,53 +33,68 @@ describe('Export Selectors', () => {
       dispatchHelper.dispatchEpisodeDownloads(store);
     });
 
-    it('should have download exports', () => {
+    it('should have podcast download exports', done => {
```

Comment:
These async tests weren't using `done()`, and turns out some of them weren't actually passing, whoops fixed!